sdk/eventhub/event-hubs/samples/getHubRuntimeInfo.ts

```diff
@@ -1,24 +1,25 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+/*
+  This sample demonstrates how to get Event Hubs information.
+*/
 
 import { EventHubClient } from "../src";
-import dotenv from "dotenv";
-dotenv.config();
 
-const connectionString = "EVENTHUB_CONNECTION_STRING";
-const entityPath = "EVENTHUB_NAME";
-const str = process.env[connectionString] || "";
-const path = process.env[entityPath] || "";
+// Define connection string and related Event Hubs entity name here
+const str = "";
+const path = "";
```

Comment:
Rename to `connectionString` and `eventHubsName` respectively. Same for all other samples as well